spark/dl/src/main/scala/com/intel/analytics/bigdl/example/loadmodel/DatasetUtil.scala

```diff
@@ -102,12 +103,26 @@ object ResNetPreprocessor {
       BGRImgToBatch(batchSize)
   }
 
-  def rdd(path: String, batchSize: Int, sc: SparkContext)
+  def rdd(path: String, batchSize: Int, sc: SparkContext, modelType : ModelType = TorchModel)
   : RDD[Sample[Float]] = {
-    val dataSet = DataSet.SeqFileFolder.filesToRdd(path, sc, classNum = 1000)
-    val transfomer = BytesToBGRImg() ->
-      BGRImgCropper(cropWidth = imageSize, cropHeight = imageSize, CropCenter) ->
-      BGRImgNormalizer(0.485, 0.456, 0.406, 0.229, 0.224, 0.225) -> BGRImgToSample()
-    transfomer(dataSet)
+    if (modelType == TorchModel) {
+      val dataSet = DataSet.SeqFileFolder.filesToRdd(path, sc, classNum = 1000)
+      val transfomer = BytesToBGRImg() ->
+        BGRImgCropper(cropWidth = imageSize, cropHeight = imageSize, CropCenter) ->
+        BGRImgNormalizer(0.485, 0.456, 0.406, 0.229, 0.224, 0.225) -> BGRImgToSample()
+      transfomer(dataSet)
+    } else if (modelType == BigDlModel) {
+      val data = DataSet.SeqFileFolder.filesToImageFrame(path, sc, 1000)
```

Comment:
different model different prerpocessing?

Reply:
Above one is from Resnet, which is quite different

Reply:
Above one is from Torch, which is quite different